src/main/java/org/folio/rest/impl/RequestType.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.rest.impl;
+
+import java.util.Arrays;
+
+import static org.apache.commons.lang3.StringUtils.equalsIgnoreCase;
```

Comment:
See the comment the `ItemStatus` enum. It applies here as well.

Reply:
done